### PR TITLE
Document known metadata keys in the schema

### PR DESCRIPTION
**Pull request by bergie**

### CODE REVIEW

File: schema/yaml/shared.yml [hunk 1]
```diff
@@ -29,6 +29,30 @@ message:
...
+  additionalProperties: true
+  required: []
+  properties:
+    x:
+      type: integer
+      description: "X coordinate of a graph entity"
+    y:
+      type: integer
+      description: "Y coordinate of a graph entity"
+    route:
```
Comment (jonnor) [inline schema/yaml/shared.yml]: Not all of these are valid for all kinds of metadata? Ex: `secure` and `route` is only on edges ?

Comment (bergie) [reply schema/yaml/shared.yml]: Correct. In flowbased/fbp#76 I did different metadata definitions for different things. We could go that granular here if we want to as well

Comment (jonnor) [reply schema/yaml/shared.yml]: I think we should, it is a bit misleading right now

Comment (bergie) [reply schema/yaml/shared.yml]: OK, changed

**Pull request merged**